tests/server/api/routers/v1/test_annotations.py

```diff
@@ -308,7 +308,7 @@ def assert_summaries(
     ) -> Callable[[bool, float], Awaitable[None]]:
         async def _(exist: bool, score_offset: float = 0) -> None:
             expected = mean_score + score_offset
-            for summaries, names in ((anno_summaries, anno_names), (eval_summaries, eval_names)):
+            for summaries, names in ((anno_summaries, anno_names), ([], eval_names)):
```

Comment:
i think you want this instead?
```suggestion
           summaries, names = anno_summaries, anno_names + eval_names
```